packages/extension-browser/src/browser-action/browser-action.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+    <meta charset="utf-8">
+    <title>Run webhint from the devtools</title>
+    <style>
+        body {
+            background-color: #4700a3;
+            color: #fff;
+            width: 700px;
+            padding: 30px;
+            text-align: center;
+        }
+        p {
+            font-size: 1.5em;
+        }
+        .logo {
+            width: 400px;
+        }
+    </style>
+</head>
+
+<body>
+    <img class="logo" src="logo.svg" alt="webhint">
+    <p>Open DevTools and navigate to the Hints tab to start using webhint.</p>
```

Comment:
Can we add F12 just to be extra clear about how to open DevTools? e.g., "Open DevTools (F12)"

Reply:
From conversations I had the F12 seems to come from the IE days and not everybody knows what it is. E.g.: I'm used to open the tools via Ctrl+Shift+I

Reply:
I'm OK with replacing (F12) with (Ctrl+Shift+I), as listed in Chrome/Edge context menu. I just think DevTools isn't going to be descriptive enough, as I see a lot of people call them things like "inspector," "elements tool," "source tool," "dev console," etc. This just makes it crystal clear to the user what we mean by DevTools and how to open them.

Reply:
But Ctrl+Shift+I is only if I'm on Windows... Other people might just do "Inspect element". There's no definitive name and having the image I think helps clarify where they should be looking at.
If they don't know what that is, they have probably download the wrong extension.

Some previous art from other extensions:

axe:
![image](https://user-images.githubusercontent.com/606594/65809273-b281e000-e150-11e9-906f-10e6dfea5c61.png)

react:
![image](https://user-images.githubusercontent.com/606594/65809288-db09da00-e150-11e9-939d-b20f47fd7d94.png)